src/sst/core/main.cc

```diff
@@ -703,6 +702,18 @@ main(int argc, char* argv[])
         return 0;
     }
 
+    // Set up the Filesystem object with the output directory
+    bool out_dir_okay = Simulation_impl::filesystem.setBasePath(cfg.output_directory());
+    if ( !out_dir_okay ) {
+        fprintf(
+            stderr,
+            "ERROR: Directory specified with --output-directory (%s) is not valid.  Most likely causes are that the "
+            "user "
+            "does not have permissions to write to this path, or a file of the same name exists.\n",
```

Comment:
You don't support overwriting a file?

Reply:
This is trying to create a directory that has the same name as a file.  If you use the fopen() or of of ofstream() calls, then you can overwrite a file.